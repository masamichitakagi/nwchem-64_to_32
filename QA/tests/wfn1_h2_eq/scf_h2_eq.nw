echo 
start scf_h2_dat

geometry
  h 0 0 -0.366269
  h 0 0  0.366269
end
basis 
  * library 6-31g**
end

task scf energy

