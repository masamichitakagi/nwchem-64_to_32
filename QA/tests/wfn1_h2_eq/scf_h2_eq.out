 argument  1 = scf_h2_eq.nw



============================== echo of input deck ==============================
echo
start scf_h2_dat

geometry
  h 0 0 -0.366269
  h 0 0  0.366269
end
basis
  * library 6-31g**
end

task scf energy

================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.6
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = LDC-153112.home
    program         = ../../../bin/MACX64/nwchem
    date            = Fri May 26 22:29:55 2017

    compiled        = Fri_May_26_09:27:12_2017
    source          = /Users/vandam/nwchem-wfn1-dev
    nwchem branch   = Development
    nwchem revision = 28658
    ga revision     = 10724
    input           = scf_h2_eq.nw
    prefix          = scf_h2_dat.
    data base       = ./scf_h2_dat.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 ORDER OF PRIMARY AXIS IS BEING SET TO 4
 D4H symmetry detected

          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 h                    1.0000     0.00000000     0.00000000    -0.36626900
    2 h                    1.0000     0.00000000     0.00000000     0.36626900

      Atomic Mass 
      ----------- 

      h                  1.007825


 Effective nuclear repulsion energy (a.u.)       0.7223888030

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             D4h       
 Group number             28
 Group order              16
 No. of unique centers     1

      Symmetry unique atoms

     1



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.73254


            XYZ format geometry
            -------------------
     2
 geometry
 h                     0.00000000     0.00000000    -0.36626900
 h                     0.00000000     0.00000000     0.36626900



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          6-31g**                   on all atoms 


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  h (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757

  2 S  1.61277800E-01  1.000000

  3 P  1.10000000E+00  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 h                          6-31g**                  3        5   2s1p


                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    10
  atoms           =     2
  closed shells   =     1
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./scf_h2_dat.movecs
  use symmetry    = T
  symmetry adapt  = T


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 h                          6-31g**                  3        5   2s1p


      Symmetry analysis of basis
      --------------------------

        a1g         3
        a1u         0
        a2g         0
        a2u         3
        b1g         0
        b1u         0
        b2g         0
        b2u         0
        eg          2
        eu          2


 Forming initial guess at       0.2s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -0.99646508

      Non-variational initial energy
      ------------------------------

 Total energy =      -0.847270
 1-e energy   =      -2.227930
 2-e energy   =       0.658271
 HOMO         =      -0.587130
 LUMO         =       0.177484

 Initial wavefunction

 global array: scf_init: MOs[1:10,1:10],  handle: -1000 

            1           2           3           4           5           6  
       ----------- ----------- ----------- ----------- ----------- -----------
   1       0.30438     0.16662    -0.74723     0.88710     0.00000     0.00000
   2       0.28738     1.67035     0.66997    -1.47894    -0.00000    -0.00000
   3      -0.00000    -0.00000    -0.00000    -0.00000     0.28071    -0.54034
   4       0.00000    -0.00000    -0.00000    -0.00000     0.54034     0.28071
   5       0.01627    -0.00586    -0.03898    -0.26949    -0.00000    -0.00000
   6       0.30438    -0.16662    -0.74723    -0.88710    -0.00000    -0.00000
   7       0.28738    -1.67035     0.66997     1.47894     0.00000     0.00000
   8      -0.00000     0.00000    -0.00000    -0.00000     0.28071    -0.54034
   9       0.00000     0.00000    -0.00000    -0.00000     0.54034     0.28071
  10      -0.01627    -0.00586     0.03898    -0.26949    -0.00000     0.00000

            7           8           9          10  
       ----------- ----------- ----------- -----------
   1       0.39219     0.00000    -0.00000    -1.49433
   2      -0.19262    -0.00000    -0.00000    -0.16450
   3       0.00000     0.70774    -0.51636     0.00000
   4      -0.00000    -0.51636    -0.70774     0.00000
   5      -0.65038    -0.00000    -0.00000    -1.60466
   6       0.39219    -0.00000     0.00000     1.49433
   7      -0.19262     0.00000     0.00000     0.16450
   8       0.00000    -0.70774     0.51636     0.00000
   9      -0.00000     0.51636     0.70774    -0.00000
  10       0.65038    -0.00000    -0.00000    -1.60466

      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------

  Numbering of irreducible representations: 

     1 a1g         2 a1u         3 a2g         4 a2u         5 b1g     
     6 b1u         7 b2g         8 b2u         9 eg         10 eu      

  Orbital symmetries:

     1 a1g         2 a2u         3 a1g         4 a2u         5 eu      
     6 eu          7 a1g         8 eg          9 eg         10 a2u     


 Starting SCF solution at       0.2s

 SCFTYPE=RHF 


 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


 #quartets = 1.560D+02 #integrals = 4.020D+02 #direct =  0.0% #cached =100.0%


 Integral file          = ./scf_h2_dat.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  49486
 No. of bits per label  =      8        No. of bits per value  =     64


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1       -1.1302535853  1.03D-01  9.99D-02      0.3
                 2       -1.1313329238  3.66D-03  3.50D-03      0.4
                 3       -1.1313335897  1.00D-07  9.49D-08      0.4


       Final RHF  results 
       ------------------ 

         Total SCF energy =     -1.131333589727
      One-electron energy =     -2.512978428913
      Two-electron energy =      0.659256036147
 Nuclear repulsion energy =      0.722388803038

        Time for solution =      0.2s



       Symmetry analysis of molecular orbitals - final
       -----------------------------------------------

  Numbering of irreducible representations: 

     1 a1g         2 a1u         3 a2g         4 a2u         5 b1g     
     6 b1u         7 b2g         8 b2u         9 eg         10 eu      

  Orbital symmetries:

     1 a1g         2 a2u         3 a1g         4 a2u         5 eu      
     6 eu          7 a1g         8 eg          9 eg         10 a2u     

             Final eigenvalues
             -----------------

              1      
    1   -0.5972
    2    0.2415
    3    0.7680
    4    1.3172
    5    1.9590
    6    1.9590
    7    2.7209
    8    2.9398
    9    2.9398
   10    4.5743

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    1  Occ=2.000000D+00  E=-5.972332D-01  Symmetry=a1g
              MO Center=  0.0D+00,  0.0D+00,  1.9D-17, r^2= 7.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.319470  1 H  s                  6      0.319470  2 H  s          
     2      0.273446  1 H  s                  7      0.273446  2 H  s          

 Vector    2  Occ=0.000000D+00  E= 2.414576D-01  Symmetry=a2u
              MO Center=  0.0D+00,  0.0D+00,  2.9D-16, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.739784  1 H  s                  7     -1.739784  2 H  s          

 Vector    3  Occ=0.000000D+00  E= 7.680493D-01  Symmetry=a1g
              MO Center=  0.0D+00,  0.0D+00,  1.9D-17, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.743426  1 H  s                  6      0.743426  2 H  s          
     2     -0.677280  1 H  s                  7     -0.677280  2 H  s          

 Vector    4  Occ=0.000000D+00  E= 1.317157D+00  Symmetry=a2u
              MO Center=  0.0D+00,  0.0D+00,  1.8D-16, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.398025  1 H  s                  7     -1.398025  2 H  s          
     1     -0.875283  1 H  s                  6      0.875283  2 H  s          
     5      0.289616  1 H  pz                10      0.289616  2 H  pz         

 Vector    5  Occ=0.000000D+00  E= 1.958996D+00  Symmetry=eu
              MO Center=  1.3D-17, -1.1D-17, -4.5D-16, r^2= 4.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.461391  1 H  px                 8      0.461391  2 H  px         
     4     -0.397348  1 H  py                 9     -0.397348  2 H  py         

 Vector    6  Occ=0.000000D+00  E= 1.958996D+00  Symmetry=eu
              MO Center= -2.3D-21, -2.7D-21, -1.4D-17, r^2= 4.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.461391  1 H  py                 9      0.461391  2 H  py         
     3      0.397348  1 H  px                 8      0.397348  2 H  px         

 Vector    7  Occ=0.000000D+00  E= 2.720886D+00  Symmetry=a1g
              MO Center= -1.3D-17,  1.1D-17, -1.4D-16, r^2= 5.6D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.650612  1 H  pz                10     -0.650612  2 H  pz         
     1     -0.387393  1 H  s                  6     -0.387393  2 H  s          
     2      0.187264  1 H  s                  7      0.187264  2 H  s          

 Vector    8  Occ=0.000000D+00  E= 2.939845D+00  Symmetry=eg
              MO Center=  2.0D-19, -1.4D-19,  4.7D-16, r^2= 5.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.707741  1 H  px                 8     -0.707741  2 H  px         
     4     -0.516362  1 H  py                 9      0.516362  2 H  py         

 Vector    9  Occ=0.000000D+00  E= 2.939845D+00  Symmetry=eg
              MO Center=  3.1D-18,  4.3D-18,  5.6D-17, r^2= 5.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.707741  1 H  py                 9     -0.707741  2 H  py         
     3      0.516362  1 H  px                 8     -0.516362  2 H  px         

 Vector   10  Occ=0.000000D+00  E= 4.574321D+00  Symmetry=a2u
              MO Center= -3.3D-18, -4.2D-18,  1.1D-16, r^2= 5.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      1.601158  1 H  pz                10      1.601158  2 H  pz         
     1      1.505806  1 H  s                  6     -1.505806  2 H  s          
     2      0.151910  1 H  s                  7     -0.151910  2 H  s          


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           0.965635269614           0.000000000000           0.000000000000
           0.000000000000           0.965635269614           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 H    1     1.00   0.51  0.48  0.01
    2 H    1     1.00   0.51  0.48  0.01

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0     -0.000000      0.000000      2.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1     -0.000000      0.000000      0.000000

     2   2 0 0     -1.524016      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -1.524016      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -1.130289      0.000000      0.958138


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.3s     wall:        0.3s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  173      173     1919      762     1056        0        0       89     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             3.15e+05 1.36e+05 2.10e+05 0.00e+00 0.00e+00 7.12e+02
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 10400 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        18	        14
	current total bytes		         0	         0
	maximum total bytes		   1054152	  22509224
	maximum total K-bytes		      1055	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
       S. A. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, V. Konjkov, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson,
      R. J. Harrison, J. R. Rehr, M. Dupuis, D. Silverstein, D. M. A. Smith,
            J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.4s     wall:        0.4s
